Wikipedia/Code/VanishAccountContentView.swift

```diff
@@ -0,0 +1,195 @@
+                                .padding([.leading, .trailing], 20)
+                            Divider().padding([.leading], 20)
+                            Text(LocalizedStrings.additionalInformationFieldTitle)
+                                .foregroundColor(Color(theme.colors.link))
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                                .font(Font(fieldTitleFont))
+                                .padding([.leading, .trailing], 20)
+                                .padding([.top], 5)
+                            TextView(placeholder: LocalizedStrings.additionalInformationFieldPlaceholder, theme: theme, text: $userInput.text)
+                                .padding([.leading, .trailing], 20)
+                                .frame(maxWidth: .infinity, minHeight: 100)
+                            Spacer()
+                                .frame(height: 12)
+                        }
+                        .background(Color(theme.colors.paperBackground))
+                        .frame(maxWidth: .infinity, minHeight: 300)
+                        VStack {
+                            VanishAccountFooterView()
+                                .foregroundColor(Color(theme.colors.secondaryText))
+                                .fixedSize(horizontal: false, vertical: true)
+                                .font(Font(bodyFont))
+                                .padding(20)
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                            Spacer()
+                            Button(action: {
+                                openMailClient()
+                            }, label: {
+                                Text(LocalizedStrings.buttonText)
+                                    .font(Font(buttonFont))
+                                    .foregroundColor(Color(theme.colors.link))
+                                    .padding()
+                                    .frame(minWidth: 335)
+                                    .frame(height: 46)
+                                    .background(Color(theme.colors.paperBackground))
+                                    .cornerRadius(8)
+                                    .padding()
+                            })
+                            Spacer()
+                        }
+                    }
+                    .padding([.bottom], extraBottomPaddingiOS13)
+                    .frame(minHeight: proxy.size.height)
+                    .frame(maxWidth: .infinity)
+                }
+                .background(Color(theme.colors.baseBackground).edgesIgnoringSafeArea(.all))
+            }
+            .onReceive(NotificationCenter.default.publisher(for: UIApplication.willEnterForegroundNotification)) { _ in
+                if shouldShowModal {
+                    withAnimation(.linear(duration: 0.3)) {
+                        toggleModalVisibility.toggle()
```

Comment:
@mazevedofs I usually love that toggle() boolean method but in this particular case I think it'll be clearer if we rename and adjust the code so that it says:

```
isModalVisible = true
shouldShowModalUponForeground = false
```